pkg/graphql/client.go

```diff
@@ -66,6 +68,11 @@ func (c *ClientImpl) ExecuteQuery(ctx context.Context, query string, variables m
 
 	res, err := c.httpClient.Do(req)
 	if err != nil {
+		if urlErr, ok := err.(*url.Error); ok {
```

Comment:
Can you please use `errors.As()` instead? There are examples here
https://golang.org/src/errors/errors.go

Reply:
Oh cool

Reply:
Done